RELEASE.md

```diff
@@ -186,6 +186,8 @@ Note, when releasing from master, all the below mentions of "release branch" mea
     ./hack/release.sh $VERSION $BRANCH
     ```
     It will prompt you whether to push it to release branch. Press `y` and hit `Enter`.
+    
+    First time running this command may need to setup the PATH environmental variable to include XXX/go/bin.  Potential location of the XXX/go/bin are /usr/local/go/bin, ~/go/bin.  Once found, run ```export PATH=$PATH:~/go/bin``` before running ```release.sh```.
```

Comment:
I think we should better link to go official documentation about why we need to do this.

Something I found:
```
$ go env
```
returns current env vars for golang.

`$GOPATH/bin` is used to store fetched binaries and default value for `$GOPATH` is `~/go`.

https://golang.org/cmd/go/#hdr-GOPATH_and_Modules